subprojects/launcher/src/integTest/groovy/org/gradle/launcher/continuous/SimpleJavaContinuousIntegrationTest.groovy

```diff
@@ -253,6 +253,8 @@ class SimpleJavaContinuousIntegrationTest extends AbstractContinuousIntegrationT
     }
 
     def "creation of initial source file triggers build"() {
+        file("src/main/java").createDir()
```

Comment:
We need this now because we don't want the creation of the _directory_ to trigger the build, right?

Reply:
Looks like we don't need this anymore: The first build creates the classes directory, so we snapshot it. That means we have a non-missing snapshot for the non-hierarchical watchers and on Linux, we watch the parent of the missing snapshot anyway. So creating the initial directory causes a file event and we trigger a new build.